iroh-base/src/hash.rs

```diff
@@ -62,6 +62,18 @@ impl AsRef<[u8]> for Hash {
     }
 }
 
+impl Borrow<[u8]> for Hash {
```

Comment:
This is unrelated, but found it useful and I don't see anything wrong with having these.